foodsaving/users/templates/activation-mail.html

```diff
@@ -1,3 +1,5 @@
+{% load i18n %}
+{% blocktrans with display_name=user.display_name %}
```

Comment:
We definitely need to split up this template into a base (containing the skeleton) and a content part (containing the message), otherwise our translators would have to copy and selectively modify all the HTML in Transifex.